projects/ion/src/lib/chip-group/chip-group.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { BadgeType } from '../badge/badge.component';
+import { ChipSize, IconDirection, IonChipProps } from '../chip/chip.component';
+
+export interface ChipInGroup extends IonChipProps {
+  selected?: boolean;
+}
+
+export interface ChipGroupProps {
+  chips?: Array<IonChipProps>;
+  disabled?: boolean;
+  size?: ChipSize;
+  multiple?: boolean;
+  events?: EventEmitter<ChipEvent>;
+  selected: EventEmitter<ChipInGroup>;
+}
+
+interface ChipEvent {
+  selected: boolean;
+  disabled: boolean;
+}
+
+interface RightBadge {
+  label: string;
+  type: BadgeType;
+}
+
+@Component({
+  selector: 'ion-chip-group',
+  templateUrl: './chip-group.component.html',
+  styleUrls: ['./chip-group.component.scss'],
+})
+export class ChipGroupComponent {
+  @Input() chips: ChipInGroup[];
+  @Input() size?: ChipSize = 'sm';
+  @Input() infoBadge?: IonChipProps['infoBadge'];
+  @Input() iconPosition?: IconDirection = 'left';
+  @Input() rightBadge?: RightBadge;
+  @Input() disabled = false;
+  @Input() multiple = false;
+  @Output() selected = new EventEmitter<ChipInGroup>();
+
+  selectChip(chipSelected: ChipInGroup): void {
+    if (chipSelected.options) {
+      this.clearChipsWithDropdown();
+    }
+    if (!this.multiple) {
+      this.clearChips();
+    }
+    chipSelected.selected = true;
+    this.selected.emit(chipSelected);
+  }
+
+  private clearChips(): void {
+    this.chips.forEach((chip) => {
+      chip.selected = false;
+    });
+  }
+
+  private clearChipsWithDropdown(): void {
+    this.chips.forEach((chip) => {
+      chip.selected && chip.options ? (chip.selected = false) : '';
```

Comment:
```suggestion
      chip.selected && chip.options && (chip.selected = false)';
```

check this.

Reply:
Line 62 serves to clean only the chips that have a dropdown, so the ternary checks if the selected chip has options(dropdown), if it does, it is deselected. If not, nothing happens.